web/content/js/versions.js

```diff
@@ -178,6 +182,14 @@ const navVersions = [
           {
             "title": "Application Centric Deployment",
             "path": "Deploying_Hosts_AppCentric"
+          },
```

Comment:
Change itself looks good IMHO. Could you please move both items to line 126?

Reply:
Addressed :+1:
Out of my curiosity, I would like to learn how we design the sequence.